### PR TITLE
[CLC-263] Configuration file format reference

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/ROOT/pages/configuration-format.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+|cert-path
+|Certificate path.
+|
+
+|key-path
+|Key certificate path.
+|
+
+|key-password
+|Key certificate path.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/configuration-format.adoc:80]: Key certificate password?

File: docs/modules/ROOT/pages/configuration-format.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+|
+
+|server
+|Server name if it's different from the certificate.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/configuration-format.adoc:84]: Server name, if different from the certificate [name ?]

File: docs/modules/ROOT/pages/configuration-format.adoc [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+|
+
+|skip-verify
+|Disables certificate verification. Shouldn't be used in production.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/configuration-format.adoc:88]: Never use in a production environment.

File: docs/modules/ROOT/pages/configuration-format.adoc [hunk 4]
```diff
@@ -0,0 +1,94 @@
+= Configuration Format
+:description: The Hazelcast CLC recognizes the following items in the configuration file.
+
+{description}
+
+The configuration is in YAML format.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/configuration-format.adoc:6]: Make this a list:

* The configuration is in YAML format.
* The configuration contains the `cluster` and `ssl` sections.
* A typical configuration file is as follows:
+
yaml
:

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/configuration-format.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/ROOT/pages/configuration-format.adoc]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4

File: docs/modules/ROOT/pages/configuration-format.adoc (hunk 2 above)
Comment (yuce) [reply docs/modules/ROOT/pages/configuration-format.adoc]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4

File: docs/modules/ROOT/pages/configuration-format.adoc (hunk 3 above)
Comment (yuce) [reply docs/modules/ROOT/pages/configuration-format.adoc]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4

File: docs/modules/ROOT/pages/configuration-format.adoc (hunk 4 above)
Comment (yuce) [reply docs/modules/ROOT/pages/configuration-format.adoc]: Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4

**Review by rebekah-lawrence — APPROVED** [review]
Looks good, thank you

**Pull request merged**